domain/src/main/kotlin/no/nav/su/se/bakover/domain/grunnlag/Grunnlag.kt

```diff
@@ -278,9 +319,29 @@ sealed class Grunnlag {
             fun List<Bosituasjon>.perioder(): List<Periode> {
                 return map { it.periode }.reduser()
             }
+
+            fun List<Bosituasjon>.perioderMedEPS(): List<Periode> {
+                return filter { it.harEPS() }.map { it.periode }.reduser()
+            }
+
+            fun List<Bosituasjon>.perioderUtenEPS(): List<Periode> {
+                return filter { !it.harEPS() }.map { it.periode }.reduser()
+            }
+
+            fun List<Bosituasjon>.harOverlappende(): Boolean {
+                return map { it.periode }.harOverlappende()
+            }
+
+            /**
+             * Kan være tom under vilkårsvurdering/datainnsamlings-tilstanden.
+             * Kan være maks 1 i alle andre tilstander.
```

Comment:
Er dette fremdeles gjeldende? Søknadsbehandling?

Reply:
Litt underlig kommentar for en fun som gir true/false for har eps, regner med det er snakk om antall eps - i så fall er den ikke lenger relevant.